frontend/src/main/web/src/components/Overlay/index.jsx

```diff
@@ -0,0 +1,81 @@
+import React, { cloneElement, PropTypes } from 'react'
+import BaseOverlay from 'react-overlays/lib/Overlay'
+import { elementType } from 'react-prop-types'
+
+const Overlay = ({
+  children,
+  transition,
+  ...props
+}) => {
+  const child = transition
+    ? children
+    : cloneElement(children, {
+      className: 'Op(1) ' + children.props.className
+    })
+  return (
+    <BaseOverlay
+      {...props}
+      transition={transition}
+    >
+      {child}
+    </BaseOverlay>
+  )
+}
+
+Overlay.propTypes = {
+  ...BaseOverlay.propTypes,
+  children: PropTypes.node,
+  transition: PropTypes.element,
+  /**
+   * Set the visibility of the Overlay
+   */
+  show: PropTypes.bool,
+  /**
+   * Specify whether the overlay should trigger
+   * onHide when the user clicks outside the overlay
+   */
+  rootClose: PropTypes.bool,
+  /**
+   * A Callback fired by the Overlay when it wishes to be hidden.
+   */
+  onHide: PropTypes.func,
+  /**
+   * Use animation
+   */
+  animation: PropTypes.oneOfType([
+    PropTypes.bool,
+    elementType
```

Comment:
This needs some explanation in its comment. I can understand the boolean value to be passed in, but what happens if something that is an `elementType` is passed in?